### PR TITLE
Add edit button to GFI toolbar

### PR DESCRIPTION
**Pull request by LukasLohoff**
- allows to switch to the edit feature form directly from GFI

image

Bugfixes:

- sets formDirty when geometry is modified
- skip exit confirmation in edit feature form if form is not dirty
- minor fixes

@terrestris/devs Please review

### CODE REVIEW

File: src/components/ToolMenu/FeatureInfo/PaginationToolbar/index.tsx [hunk 1]
```diff
@@ -86,6 +121,41 @@ export const PaginationToolbar: React.FC<PaginationToolbarProps> = ({
...
+    if (!layer || !map) {
+      return;
+    }
+    const selectedFeatureClone = selectedFeature.clone();
+    const geojsonFeatureString = new OlFormatGeoJSON().writeFeature(selectedFeatureClone);
+
+    try {
+      const geojsonFeature = JSON.parse(geojsonFeatureString) as Feature;
+      const editLayer = DigitizeUtil.getDigitizeLayer(map);
+      if (editLayer) {
```
Comment (dnlkoch) [inline src/components/ToolMenu/FeatureInfo/PaginationToolbar/index.tsx:134]: Absolutely minor, but inverting the conditions (and returning) might increase readability.

**Review by dnlkoch — APPROVED** [review]
Very nice @LukasLohoff!

**Comment by LukasLohoff** [comment]
Thanks for the review!

**Pull request merged**